Makefile

```
@@ -0,0 +1,32 @@
+browser-extensions:
+	npm run build
+
+check-apikey:
+		curl '$POLYSWARM_API_URL/v2/instance' \
+		  -H 'authority: $POLYSWARM_API_URL' \
```

```suggestion
		curl "${POLYSWARM_API_URL}/v2/instance" \
		  -H "authority: ${POLYSWARM_API_URL}" \
```

I don't think this is going to work, `$VAR` only works between strings surrounded of `"` quotes, not single `'` quotes. Moreover, Make don't like the use of env variables without surrounding them with `${}` or `$()` as well.

---

I see. I did this as a quick refactor, but expected CI to be running before running full tests.

Will commit the suggestion and test locally for now. Thanks for reporting